app/assets/stylesheets/layout/common/_header.scss

```diff
@@ -1,7 +1,7 @@
 .l-header {
-  background: $color-green-primary;
```

Comment:
Problem here is that the inner element is what is sticky whilst the header itself scrolls off the viewport in this scenario, which we implemented to retain the depth of the header when its position is fixed.